tap_spotify/streams.py

```diff
@@ -162,3 +162,14 @@ class GlobalViralTracksDailyStream(_PlaylistTracksStream):
     name = "global_viral_tracks_daily_stream"
     path = "/playlists/37i9dQZEVXbLiRSasKsNU9"
     primary_keys = ["rank", "synced_at"]
+
+
+class UserSavedTracksStream(SpotifyStream):
```

Comment:
```suggestion
class UserSavedTracksStream(_SyncedAtStream, SpotifyStream):
```

This stream needs to inherit the logic required for use of `synced_at`.